src/screens/Accounts/AccountRow.js

```diff
@@ -7,7 +7,7 @@ import { compose } from "redux";
 import { connect } from "react-redux";
 import { createStructuredSelector } from "reselect";
 import { listSubAccounts } from "@ledgerhq/live-common/lib/account";
-import { listTokenTypesForCryptoCurrency } from "@ledgerhq/live-common/src/currencies";
+import { listTokenTypesForCryptoCurrency } from "@ledgerhq/live-common/lib/currencies";
```

Comment:
holy shit, we need a script/rule to detect these.

Reply:
agreed